tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "module": "amd",
     "moduleResolution": "node",
-    "target": "es5",
+    "target": "es6",

```

Comment:
Zach,
I'm not sure any more this has been a while ago. It could be that I was changing things by trial and error. 

As far as I can remember there were 2 different tsconfig specified by in the project. One in the root and one in the doc folder (which was explicitly used by the document generation step in the build). If I remember correct generating the documentation was also causing some errors. So I merged these settings together (opting for the most modern setting) and pointed the doc generation step to the root tsconfig.

Looking up the meaning of the settings in https://www.typescriptlang.org/docs/handbook/compiler-options.html. It doesn't reveal to much but I guess it will be the output version of the JavaScript generated by the tsc. Checking the http://caniuse.com/#search=es6 this means that IE11 is not supported.

I guess it would be better to reset it back to ES5?

To make your decision a bit harder ;-) I added an issue link to another aurelia project (bootstrap) which is using the "es2015" (which is the official es6 name). https://github.com/aurelia/bootstrapper/issues/42. We could opt to align this setting with other au projects?

Let me know what you want it to be.  

Kind Regards
Dries